### ISSUE TITLE
Enhancing the textual formatting and spacing within the website to optimize its presentation and readability.

### ISSUE THREAD

**Issue by samSoumyajitD** [arakoodev/doc#14]
### Is your feature request related to a problem? Please describe.

Hello @arakoodev I  m a contributor of Gssoc'23

 I have been exploring the website and have noticed an opportunity for improvement. There appears to be an abundance of textual content that, in my opinion, requires some attention. I believe implementing proper text formatting, along with adequate spacing, would greatly enhance the overall readability of the website. Additionally, I would recommend considering subtle adjustments to the color combination while maintaining the existing theme, as it would further contribute to an enhanced reading experience.

I kindly request you to assign me this particular issue under the label GSSOC'23 as I am eager to make a meaningful contribution to this project.

### Describe the solution you'd like

Proper text formatting, along with adequate spacing, would greatly enhance the overall readability of the website. Additionally, I would recommend considering subtle adjustments to the color combination while maintaining the existing theme, as it would further contribute to an enhanced reading experience.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samSoumyajitD** [comment]
Please assign this issue to me under GSSoC'23 @sandys

**Comment by sandys** [comment]
@samSoumyajitD we have pushed a new design to the website based on other work. please check and see if u still want to do this bug fix. if yes, please modify bug with new screenshots and proposal and i will assign this bug to you.

**Comment by samSoumyajitD** [comment]
@sandys yes sir i want to pls give me two more days n i will fix the bug.
